traffic_ops/app/lib/UI/ConfigFiles.pm

```diff
@@ -475,9 +475,14 @@ sub ip_allow_data {
 	$ipallow->[$i]->{action} = 'ip_allow';
 	$ipallow->[$i]->{method} = "ALL";
 	$i++;
+
+	# default for coalesce_ipv4 = 24, 5 and for ipv6 48, 5; override with the parameters in the server profile.
```

Comment:
maybe just get rid of the comment? it doesn't look like it's accurate with the code below